oxenc/bt_producer.h

```diff
@@ -80,8 +80,7 @@ namespace detail {
                     std::forward<AppendArgs>(app_args)...,
                     std::string_view{reinterpret_cast<const char*>(sig.data()), sig.size()});
         } else
-            static_assert(
-                    false,
+            throw std::invalid_argument(
```

Comment:
Errors detectable at compile time should do so, rather than be runtime errors.  If the `false` is causing trouble (apparently it's standard-compliant only from C++23 onwards), perhaps `std::is_void_v<SignFunc>` will work.